tests/utils_math/unittest.cpp

```diff
@@ -288,20 +288,19 @@ TEST_F(Saturate2dVector, ValsOnEdgeOfRange) {
    float inputVal_x;
    float inputVal_y;
    float inputVal_max;
-   bool ret;
 
    inputVal_x = 1.0;
    inputVal_y = 1.0;
    inputVal_max = sqrtf(2);
-   ret = utils_saturate_vector_2d(&inputVal_x, &inputVal_y, inputVal_max);
+   utils_saturate_vector_2d(&inputVal_x, &inputVal_y, inputVal_max);
```

Comment:
I don't know that we should change this, since in the other cases we check that the returned variable is correct. Here we don't, but there's a comment to this effect a few lines down and that comment would be dangling if `ret` were removed.